lib/Cro/HTTP/Client/CookieJar.pm6

```diff
@@ -49,6 +49,32 @@ monitor Cro::HTTP::Client::CookieJar {
         False
     }
 
+    method !checkCookieAge(Cro::HTTP::Cookie $cookie) {
```

Comment:
We consistently use kebab case for method names, so this would be more fitting as `check-cookie-age`. Also it's not really checking so much as figuring things out; I'd maybe call it `get-cookie-lifetime` instead.

Reply:
Sure.

Reply:
Changes have been made, as requested.